nidmm system test test_vi_string_attribute fails with NI-DMM 2023Q3 and later

<!--
When filing an issue, please fill in the requested fields as thoroughly as you can and then delete this comment
Note: For issues, assistance, and questions about using NI products, please visit ni.com/support. Issues opened here should be specifically related to these Python APIs.
-->

### Description of issue

Due to a change in the reported serial for a simulated DMM, the nidmm system test `test_vi_string_attribute` fails with the NI-DMM 2023Q3 runtime and later.

```
AssertionError: assert '0' == 'FFFFFFFF'
  - FFFFFFFF
  + 0
```

### Steps to reproduce issue
Run the system tests with NI-DMM 2023Q3 (23.5.0) runtime.

--- a/src/nidmm/system_tests/test_system_nidmm.py
+++ b/src/nidmm/system_tests/test_system_nidmm.py
@@ -42,9 +42,9 @@ class SystemTests:
 
     # Attribute tests
     def test_vi_string_attribute(self, session):
-        assert session.serial_number == 'FFFFFFFF'
+        assert session.instrument_model == 'NI PXIe-4082'
         try:
-            session.serial_number = 'FFFFFFFA'
+            session.instrument_model = 'NI PXIe-4081'
         except nidmm.Error as e:
             assert e.code == -1074135027  # Attribute is read-only
 

